### DISCUSSION TITLE
How to add token statistics

### DISCUSSION THREAD

**Discussion by E8A281E6ACA2** [open-webui/open-webui#2230] [General]
This is a great ui, I hope the devs add visualizations that list things like token usage. Thanks.

**Comment by rober-tm** [comment]
Justamente estaba buscando lo mismo, en la hoja de ruta tiene planeado esto?
Porque veo que en los modelos ejecutados con Ollama si lo tiene, el costo de tokens, cuando se utiliza la API de openAi no apararece esa informacion.

**Reply by xrwang8** [reply]
What is the current status？
